src/work-allocation/containers/task-manager-list/task-manager-list.component.html

```diff
@@ -6,6 +6,6 @@
 </p>
 <exui-task-list
   [taskServiceConfig]="taskServiceConfig" [tasks]="tasks" [fields]="fields"
-  [sortedBy]="sortedBy" [emptyMessage]="emptyMessage"
+  [sortedBy]="sortedBy" [emptyMessage]="emptyMessage" [sourceColumn]="sourceColumn" [matchValue]="matchValue"
   (sortEvent)="onSortHandler($event)" (actionEvent)="onActionHandler($event)">
 </exui-task-list>
```

Comment:
Linting will fail without a line break at the end.

Reply:
I've just realised you might be getting a lot of emails because of my comments so I'm just going to say 'all above resolved' in future unless there's a debate or something